src/applications/personalization/profile/mocks/server.js

```diff
@@ -18,7 +18,9 @@ const responses = {
   'GET /v0/ppiu/payment_information': paymentHistory,
   'POST /v0/profile/address_validation': address.addressValidation,
   'GET /v0/mhv_account': mhvAcccount,
-  'GET /v0/profile/personal_information': personalInfo,
+  'GET /v0/profile/personal_information':
+    personalInformation.getBasicUserPersonalInfo,
+  ...personalInformation.putPreferredName,
```

Comment:
THINKING OUT LOUD: Cool idea! I am curious how this would look scaled to other endpoints. I like to have `server.js` as a list of all the endpoints, but I definitely see the value in something like this.

Reply:
I totally get the idea of having all the endpoints listed in a single entrypoint. Maybe I can morph this a little to still list the endpoint but also still be able to chunk things out a little. Ill think about this and see what I can come up with! Thanks for the feedback :)